prime-router/docs/playbooks/how-to-use-filters.md

```diff
@@ -77,6 +77,7 @@ filters:
   - topic: "covid-19"
     jurisdictionalFilter: [ "orEquals(ordering_facility_state, NJ, patient_state, NJ)" ]
     qualityFilter:  [ "hasAtLeastOneOf(order_test_date,specimen_collection_date)" , etc, etc]
+    conditionFilter: ["%conditions.intersect('600-7').exists()"]**
```

Comment:
This will not apply to a covid 19 filter. Remove. I would not update this at all since we have to build complete documentation for the UP.